stable/datadog/values.yaml

```diff
@@ -12,6 +12,17 @@ image:
 # get guaranteed delivery of the metrics in Datadog-per-namespace setup!
 daemonset:
   enabled: true
+  ## Bind ports on the hostNetwork. Useful for CNI networking where hostPort might
```

Comment:
Could you add the same warning as for hostPort here? Also it's for custom metrics and traces (8126 is for the trace agent)